src/doc/src/guide/continuous-integration.md

```diff
@@ -21,6 +21,42 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+### GitHub Actions
+
+To test your package on GitHub Actions, here is a sample `.github/workflows/ci.yml` file:
+
+```yaml
+name: Cargo Build & Test
+
+on:
+  push:
+
+jobs:
+  build_and_test:
+    name: Rust project - latest
+    runs-on: ${{ matrix.operating-system }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain:
+          - stable
+          - beta
+          - nightly
+        operating-system:
+          - "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
```

Comment:
Historically we've tried to avoid third-party recommendations in Rust documentation. I think we could instead recommend doing:

```
- run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
```